src/main/java/io/cdap/plugin/gcp/bigtable/sink/BigtableSinkConfig.java

```diff
@@ -72,12 +72,13 @@
   private final String bigtableOptions;
 
   public BigtableSinkConfig(String referenceName, String table, String instance, @Nullable String project,
-                            @Nullable String serviceFilePath, String keyAlias, String columnMappings,
-                            @Nullable String bigtableOptions) {
+                            @Nullable String serviceAccountType, @Nullable String serviceFilePath, String keyAlias,
+                            String columnMappings, @Nullable String bigtableOptions) {
     this.referenceName = referenceName;
     this.table = table;
     this.instance = instance;
     this.project = project;
+    this.serviceAccountType = serviceAccountType;
```

Comment:
where is serviceAccountType used?

Reply:
> where is serviceAccountType used?

this was required by tests - in order to make existing tests work.